org.eclipse.scout.sdk.s2i/src/main/kotlin/org/eclipse/scout/sdk/s2i/widgetmap/WidgetMapUpdater.kt

```diff
@@ -143,6 +143,7 @@ object WidgetMapUpdater {
 
         val operation = WidgetMapCreateOperation()
         operation.setLiteral(objectLiteral.api())
+        operation.setMainWidget(consumerClass?.api())
```

Comment:
Now as you pass the modelConsumer, there is no need to pass isPage anymore as the operation can compute it based on the consumer. So WidgetMapUpdater#isPageModel could be moved to the operation and the setter can be protected instead?